### PR TITLE
feat(pci.container): merge the S3 standard & high-perf endpoints

### PR DESCRIPTION
**Pull request by brofik**
ref: TAPC-769

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-769 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related
- [ ] CMT-12911

### CODE REVIEW

**Review by selm3n — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/object/add/add.html [hunk 1]
```diff
@@ -9,6 +9,23 @@
...
+                data-ng-href="{{$ctrl.priceLink}}"
+                target="_blank"
+                rel="noopener"
+            >
+                <span
+                    data-translate="pci_projects_project_storages_containers_container_object_add_storage_class_label_description_link"
+                ></span>
+                <i
+                    class="oui-icon oui-icon-arrow-right align-middle d-inline-block"
+                ></i>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/object/add/add.html:26]:
```suggestion
                <i
                    aria-hidden="true"
                    class="oui-icon oui-icon-arrow-right align-middle d-inline-block"
                ></i>
```

File: packages/manager/modules/pci/src/projects/project/storages/containers/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -27,8 +27,7 @@
 
   "pci_projects_project_storages_blocks_add_archive_label": "Créer un conteneur d'archive",
   "pci_projects_project_storages_blocks_add_object_label": "Créer un conteneur d'objets",
-  "pci_projects_project_storages_containers_offer_high_performance": "High Performance - S3",
-  "pci_projects_project_storages_containers_offer_standard_s3": "Standard - S3",
+  "pci_projects_project_storages_containers_offer_s3": "S3",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/translations/Messages_fr_FR.json:30]: **nitpick:** Can be maintained as a constant.

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/containers/translations/Messages_fr_FR.json:30]: The translation is already done on it. 
We could do this change in a future optimisation.

**Review by anooparveti — APPROVED** [review]

**Review by selm3n — DISMISSED** [review]

**Comment by anooparveti** [comment]
Rebasing with `release/pci-w25`

**Review by anooparveti — APPROVED** [review]

**Review by selm3n — APPROVED** [review]

**Pull request merged**